OptimizelySDK.NetStandard16/OptimizelySDK.NetStandard16.csproj

```diff
@@ -57,11 +57,12 @@
         <Compile Include=".\Properties\AssemblyInfo.cs" />
         <Compile Include="..\OptimizelySDK\Utils\ConfigParser.cs" />
         <Compile Include="..\OptimizelySDK\Utils\Schema.cs" />
-    		<Compile Include="..\OptimizelySDK\Utils\ControlAttributes.cs" />
-		    <Compile Include="..\OptimizelySDK\Utils\ExceptionExtensions.cs" />
-    		<Compile Include="..\OptimizelySDK\Utils\ExperimentUtils.cs" />
-		    <Compile Include="..\OptimizelySDK\Utils\ConditionParser.cs" />
-    		<Compile Include="..\OptimizelySDK\Utils\AttributeMatchTypes.cs" />
+		<Compile Include="..\OptimizelySDK\Utils\ControlAttributes.cs" />
```

Comment:
Please indent. 
Why all these files are included in .netstandard16? Not in other Projects?